.pre-commit-config.yaml

```diff
@@ -0,0 +1,41 @@
+repos:
+  - repo: https://github.com/PyCQA/flake8
+    rev: 6.0.0
+    hooks:
+      - id: flake8
+        additional_dependencies:
+          - flake8-executable
+          - bandit
+          - flake8-bugbear
+          - flake8-comprehensions
+          - flake8-simplify
```

Comment:
These cannot be versioned, I guess?

Reply:
Unpinned them for testing, but indeed will use the versions from `lint.txt`